.github/workflows/refresh-manifests.yml

```diff
@@ -0,0 +1,39 @@
+name: Refresh Manifests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 16 * * *' # 8am PST (16:00 UTC) - same schedule as API review
```

Comment:
I think it would be better if the manifest was updated on the PRs that made them change. Not in a separate batched PR.

Reply:
It looks like a test (or GH action) that runs on PRs and fails when they are out of date.